dlgr/griduniverse/static/scripts/demo.js

```diff
@@ -246,13 +246,13 @@ Player.prototype.replaceItem = function(item) {
   if (item && !(item instanceof itemlib.Item)) {
     item = new itemlib.Item(item.id, item.item_id, item.maturity, item.remaining_uses)
   }
-  this.current_item = item;
+  this.currentItem = item;
   displayWhatEgoPlayerIsCarrying(item);
 };
 
 Player.prototype.getTransition = function () {
   var transition;
-  var player_item = this.current_item;
+  var player_item = this.currentItem;
```

Comment:
While we're making these kinds of stylistic changes, we might want to add `prettier` to our pre-commit hooks. (Partly a selfish request because I'm having a hard to time _preventing_ `prettier` from formatting files. :)